backend/lobby/consumers.py

```diff
@@ -25,7 +30,23 @@ def randomcards_deck_handler(self, data):
         cards_required = data['payload']
         self.send_server_response_to_client('GAMECONSOLE_DECISION', username, room, cards)
 
+    def get_the_three_store_cards(self):
```

Comment:
Is it possible to add a parameter so Lakshmi can request 0 or more cards?

Her message includes a cards attribute where she specifies a number of requested cards.

Reply:
I just wrote that to get the whole card store then in her handling of the result I figured she could filter what she wanted. I think getting the full card store each time would probably be easier because if you buy the middle or first card then the other slide over and the deck handler automatically draws a third card. If the draw pile runs out the deck handler automatically shuffles the discard pile etc. So just copying the full state of the card store means less logic on the frontend

Reply:
@kaubster not sure if my reply automatically pings you

Reply:
Ok, I like that approach. Sold....